website/versioned_docs/version-v1.3_alpha/troubleshooting.md

```diff
@@ -49,3 +49,113 @@ You can generate the missing `.dat` files using the following command:
 ```bash
 pnpm build:circuits-c
 ```
+
+## contracts: `prove` command failure
+
+### Case `Commitment mismatch`
+
+If your log looks like following, that's because you have already run the `prove` command before. You could either do go to `cli`, and run `genProof` command.
```

Comment:
nit: you are saying "either" so there should be another option too. Maybe just say, as you run from the contracts folder, you can try again from the `cli` using the `genProof` command (or smth like that)